docs/features/running-integration-tests.md

```diff
@@ -12,37 +12,49 @@ Easy to start:
 > Install-Package Arcus.EventGrid.Testing
 ```
 
+Here is what this article will cover:
+
+- [Receiving events in your tests](#receiving-events-in-your-tests)
+- [The Azure infrastructure](#azure-infrastructure)
+- [Test Example](#example)
+- [Troubleshooting tests](#troubleshooting-tests)
+    - [Use logging](#use-logging)
+    - [Keeping a test subscription on the topic](#keeping-a-test-subscription-on-the-topic)
+
 ### Receiving events in your tests
-By using the `HybridConnectionHost` you can subscribe to Azure Event Grid events on a custom topic and consume them in your tests. This is achieved by setting up a [Azure Relay Hybrid Connection](https://docs.microsoft.com/en-us/azure/service-bus-relay/relay-what-is-it#hybrid-connections) which is subscribing for events and storing them internally in the host.
+By using the `ServiceBusEventConsumerHost` you can subscribe to Azure Event Grid events on a custom topic and consume them in your tests.
 
 You can easily check if an event is received:
 ```csharp
-_hybridConnectionHost.GetReceivedEvent(eventId, retryCount: 5)
+_serviceBusEventConsumerHost.GetReceivedEvent(eventId, retryCount: 5)
```

Comment:
Would be good to explain what the `retryCount` actually does (with regards to how long it waits)